### DISCUSSION TITLE
create-user doesn't work

### DISCUSSION THREAD

**Discussion by alexlyzhov** [danny-avila/LibreChat#3039] [Help Wanted]
My setup is DigitalOcean and docker-compose.

From the configuration manual (https://www.librechat.ai/docs/configuration/authentication):

> Quick Tip: Even with registration disabled, add users directly to the database using npm run create-user. If you can’t get npm to work, try sudo docker exec -ti LibreChat sh first to “ssh” into the container.

`sudo docker exec -ti LibreChat sh` fails because there's no LibreChat container.

I also tried `sudo docker exec -ti LibreChat-API sh` and then run `npm run create-user`, but it doesn't provide an output regardless of whether I supply the arguments or not. This is how it looks:

```
/app/api # npm run create-user

> @librechat/backend@0.7.2 create-user
> node ./create-user.js
```

And after that there's no output.

**Comment by light299792458** [comment]
I have the same problem, did you find a solution?

**Reply by alexlyzhov** [reply]
No, I didn't.

**Comment by kukoboris** [comment]
I have output like this
``` javascript
/app/api # npm run create-user

> @librechat/backend@v0.7.5-rc1 create-user
> node ./create-user.js

node:internal/modules/cjs/loader:1228
  throw err;
  ^

Error: Cannot find module '/app/api/create-user.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1225:15)
    at Module._load (node:internal/modules/cjs/loader:1051:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12)
    at node:internal/main/run_main_module:28:49 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v20.17.0
npm error Lifecycle script `create-user` failed with error:
npm error code 1
npm error path /app/api
npm error workspace @librechat/backend@v0.7.5-rc1
npm error location /app/api
npm error command failed
npm error command sh -c node ./create-user.js
/app/api #
```

**Comment by danny-avila** [comment]
I just tried it, it's working. This is assuming you are using the `deploy-compose.yml` file:

1. Exec into the container:
   ```
   docker exec -it LibreChat-API /bin/sh
   ```

2. Navigate up one directory:
   ```
   cd ..
   ```

3. Run the create-user script:
   ```
   npm run create-user
   ```

To combine these steps into a single command that you can run from your host machine:

```
docker exec -it LibreChat-API /bin/sh -c "cd .. && npm run create-user"
```

This command will exec into the container, change to the parent directory, and run the create-user script in one go.

**Reply by kukoboris** [reply]
Wow, it is working!!! Thank you!

**Comment by danny-avila** [comment]
Closing since I've updated the documentation to go over this better:
https://www.librechat.ai/docs/configuration/authentication#user-management-scripts

**Reply by kyr0** [reply]
@danny-avila I just ran into this problem exactly with the main revision we have today. `docker-compose` is not even a thing today anymore, as it would be `docker compose` -- and we need the ".." definitely otherwise the reported error occurs. I think the docs are outdated?
